### ISSUE TITLE
Flutter example android build error

### ISSUE THREAD

**Issue by no1seman** [AdobeXD/xd-to-flutter-plugin#38]
With actual Flutter (Channel stable, v1.17.1) got build error:
```
...
FAILURE: Build failed with an exception.

* Where:
Settings file '...\xd-to-flutter-plugin\adobe_xd\example\android\settings.gradle' line: 15

* What went wrong:
A problem occurred evaluating settings 'android'.
> Could not read script '...\flutter\packages\flutter_tools\gradle\app_plugin_loader.gradle' as it does not exist.
...
```

Seems that "...\xd-to-flutter-plugin\adobe_xd\example\android\settings.gradle" must be updated because there is no longer "app_plugin_loader.gradle" gradle script avaible in latest flutter SDK.

"settings.gradle" should contain smth like this:
```
include ':app'

def flutterProjectRoot = rootProject.projectDir.parentFile.toPath()

def plugins = new Properties()
def pluginsFile = new File(flutterProjectRoot.toFile(), '.flutter-plugins')
if (pluginsFile.exists()) {
    pluginsFile.withReader('UTF-8') { reader -> plugins.load(reader) }
}

plugins.each { name, path ->
    def pluginDirectory = flutterProjectRoot.resolve(path).resolve('android').toFile()
    include ":$name"
    project(":$name").projectDir = pluginDirectory
}
```

**Comment by gskinner** [comment]
I'll try to take a look at this today and get it into the next build. If you have a chance to test your solution above, that would help a lot. Thanks!

**Comment by no1seman** [comment]
@gskinner,
thanks for you answer.

On Win10 with Flutter (Channel stable, v1.17.1) suggested above settings.gradle works fine for me.
Another way is to upgrade Flutter SDK to latest release/stable/master and then create blank flutter project using "flutter create" and then simply mergewith existed lib/ and pubspec.yaml (It would be gret if it will be also be upgraded, adobe_xd: ^0.1.0 dependency is also outdated).

**Comment by gskinner** [comment]
Thanks. Creating a new project was the route I was planning to take in order to ensure everything was fully updated.

**Comment by gskinner** [comment]
Digging into this, it looks like the issue was that the example project was created with the dev channel, and it appears to be incompatible with the stable channel. Recreating it with stable seems to have solved the problem, but I'm going to do more testing before committing.

**Comment by gskinner** [comment]
So that does seem to fix it. The issue is that stable doesn't support desktop, which makes testing significantly slower. Perhaps staying on dev, but documenting it in the README is the way to go?

Thoughts @no1seman ?

**Comment by gskinner** [comment]
Doc fix in b274c0e32990089fe38e215a3d1690a4462e0b20

**Comment by no1seman** [comment]
Well, for now Google separates examples for iOS/Android (Flutter SDK production status), Web (Flutter SDK Beta status, avaible only in dev branch) and Desktop (Flutter SDK Alfa status for MacOS, Tech preview for Win/Linux). Seems that the best way is to follow by BIG rabbit! )
